scanner.py

```diff
@@ -329,6 +329,13 @@ def __init__(self, config: ConfigWrapper):
             desc=self.cmd_Z_OFFSET_APPLY_PROBE_help,
         )
 
+    def is_model(self, gcmd: GCodeCommand):
```

Comment:
```suggestion
    def is_model_loaded(self, gcmd: GCodeCommand):
```

Reply:
We could also validate the model here. Just name the method appropriately 😄

Reply:
https://github.com/Cartographer3D/cartographer-klipper/pull/165/commits/15803433ec8a043d70c19f21589482b8cb8f6fa7